docs/usage.md

```diff
@@ -128,133 +146,119 @@ yarn run gulp test:acceptance # Run only acceptance (in-browser) tests on produc
 yarn run gulp audit           # Run code quality audits.
 ```
 
-## Usage: Docker
+## Docker
 
-Much of the guidance above for the "stand-alone" set-up still stands, and it
-is worth reviewing in full. Here are some things that might be different:
+We use [`docker-compose`](https://docs.docker.com/compose/reference/overview/)
+to run an Elasticsearch container, a PostgreSQL container, 
+and Django in Python 2.7 and 3.6 containers. 
+There is also a container serving the documentation. 
 
-- `docker-compose` takes care of running Elasticsearch for you, and all
-Elasticsearch, Postgres, and Python output will be shown in a single Terminal
-window or tab. (wherever you run `docker-compose up`)
-- `manage.py` commands can only be run after you've opened up a terminal in the
-Python container, which you can do with `./shell.sh`
-- There is not *yet* a good way to use SSL/HTTPS, but that is in the works
-- You won't ever need to use `backend.sh` or `runserver.sh`
+All of these containers are configured in our 
+[`docker-compose.yml` file](https://github.com/cfpb/cfgov-refresh/blob/master/docker-compose.yml). 
+See the [Docker documentation](https://docs.docker.com/compose/compose-file/) 
+for more about the format and use of this file.
 
-### How do I...
+The following URLs are mapped to your host from the containers:
 
-#### Use Docker Machine
+- Python 2.7 container: http://localhost:8000/
+- Python 3.6 container: http://localhost:8333/
+- Elasticsearch: http://localhost:9200/
+- This documentation: http://localhost:8888/
 
-If you used `mac-virtualbox-init.sh`, then we used Docker Machine to create a
-VirtualBox VM, running the Docker server. Here are some useful `docker-machine`
-commands:
+To build and run the containers for the first time, run:
 
-- Start and stop the VM with  `docker-machine start` and `docker-machine stop`
-- get the current machine IP with `docker-machine ip`
-- if for some reason you want to start over, `docker-machine rm default`, and
-  `source mac-virtualbox-init.sh`
-
-To enable Docker and Docker Compose commands, you'll always first need to run
-this command in any new shell:
+```bash
+docker-compose up
+```
 
-`eval $(docker-machine env)`
+### Environment variables
 
-It may be helpful to run `docker-machine env` by itself, so you understand
-what's happening. Those variables are what allows `docker-compose` and the
-`docker` command line tool, running natively on your Mac, to connect to the
-Docker server running inside VirtualBox.
+Environment variables from your `.env` file are sourced 
+when the python containers starts 
+and when you [access the running python containers](#access-the-containers-shell). 
```

Comment:
Has that always happened?! Or is that a new feature being added by the `RUN echo 'source /src/cfgov-refresh/.env' >> ~/.bashrc` line you put in the `Dockerfile`?

```suggestion
and when you [access the running Python containers](#access-the-containers-shell). 
```

Reply:
It has always happened, but instead of that echo in the `Dockerfile`, it was happening in the `shell.sh` every time it was invoked.